sphinx/deprecation.py

```diff
@@ -29,23 +29,54 @@ class RemovedInSphinx50Warning(PendingDeprecationWarning):
 RemovedInNextVersionWarning = RemovedInSphinx40Warning
 
 
-def deprecated_alias(modname: str, objects: Dict, warning: "Type[Warning]") -> None:
+def deprecated_alias(modname: str, objects: Dict[str, Tuple[str, object]],
+                     warning: "Type[Warning]") -> None:
     module = import_module(modname)
-    sys.modules[modname] = _ModuleWrapper(module, modname, objects, warning)  # type: ignore
+    sys.modules[modname] = _ModuleWrapperDeprecatedAlias(  # type: ignore
+        module, modname, objects, warning)
 
 
-class _ModuleWrapper:
-    def __init__(self, module: Any, modname: str, objects: Dict, warning: "Type[Warning]"
-                 ) -> None:
+class _ModuleWrapperDeprecatedAlias:
+    def __init__(self, module: Any, modname: str,
+                 aliases: Dict[str, Tuple[str, object]],
```

Comment:
Could you add an assertion that confirms the value of `aliases` is a tuple to this constructor? I'm afraid to pass a class object as its value as before.